### PR TITLE
[merged] syscontainers: add rollback

### PR DESCRIPTION
**Pull request by yuqi-zhang**
Add --rollback flag to atomic update, which switches a system
container to the other deployment if one exists.

Signed-off-by: Yu Qi Zhang <jerzhang@redhat.com>

### CODE REVIEW

**Comment by yuqi-zhang** [comment]
As a side note perhaps it makes sense to refractor the commands to `atomic containers update` and `atomic containers rollback` instead of being `atomic update --container` and `atomic update --rollback --container`.

**Comment by rhatdan** [comment]
Yes that makes more sense to me.  I love the idea of getting rid of base verbs also.

**Comment by giuseppe** [comment]
can we do the refactoring separately?  It doesn't depend from this PR, which I think can be already merged

**Comment by yuqi-zhang** [comment]
Sure, I am working on refactoring already based on this branch. Feel free to merge this, or I can add commits to this PR if it's better.

**Comment by giuseppe** [comment]
fine to merge this for now?  We can work on the refactoring separately, as anyway it influences the entire "update" command

File: Atomic/syscontainers.py [hunk 1]
```diff
@@ -607,14 +601,12 @@ def rollback(self, name):
         previous_deployment = 0
         if os.path.realpath(path).endswith(".0"):
             previous_deployment = 1
-        else:
-            previous_deployment = 0
 
-        if not os.path.exists("%s.%d" % (path, previous_deployment)):
-            raise ValueError("Error: Cannot find a previous deployment to rollback.")
+        destination = "%s.%d" % (path, previous_deployment)
```
Comment (rhatdan) [inline Atomic/syscontainers.py]: How about remove prvious_deployment altogether.
 destination = "%s.%d" % (path, (1 if os.path.realpath(path).endswith(".0") else 0)

**Comment by rhatdan** [comment]
LGTM except for last suggestion.

**Comment by yuqi-zhang** [comment]
Fixed and rebased

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**